src/Notepads/Notepads.csproj

```diff
@@ -443,6 +443,15 @@
   <ItemGroup>
     <PRIResource Include="Strings\**\*.resw" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
You don't need to add these since it is already covered by "<PRIResource Include="Strings\**\*.resw" />" You can remove all new lines added in this file.

Reply:
Hi, I already remove that lines, verify my last commit